AB_environments/AB_baseline.conda.yaml

```diff
@@ -5,7 +5,43 @@
 channels:
   - conda-forge
 dependencies:
+    # Copy-paste from recipe/meta.yaml, minus dask packages (see at bottom)
     - python=3.9
-    # Here we're using coiled-runtime for the sake of convenience, but we didn't have
-    # to. We could have also pointed to a nightly build.
-    - coiled-runtime=0.2.1
+    - pip
+    - coiled >=0.2.54
+    - numpy ==1.23.5
+    - pandas ==1.5.3
```

Comment:
pandas 2.0 is out on pip, probably it'll be out on conda-forge soon, should we bump this too?

Reply:
Shouldn't we wait for full integration in dask and distributed CI?

Reply:
I'm in favor of bumping to 2.0 now that it's officially out (assuming that it doesn't break CI here). The `dask` test suite is _generally_ passing with `pandas=2.0`, there are a small number of `xfail`s + follow-up issues, but largely things work with `dask.dataframe`. There's one small fix in flight for the `distributed` test suite (it's a tests-only fix).

Having `pandas=2.0` will also make it possible to use the new `dataframe.convert-string` config option in other tests here.

Reply:
See below. Can we leave the temporary removal of xarray - and bump to pandas 2 - to a separate PR?

Reply:
Sounds good 👍